samples/multicluster/expose-istiod-https.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
This needs a mention in the readme, especially since it requires manual modification

Reply:
which readme?  The doc is WIP in a google doc and will be transferred to an istio.io PR.